mlos_bench/mlos_bench/config/environments/apps/fio/fio.jsonc

```diff
@@ -60,16 +74,21 @@
                         "trial_id",
                         "fio_prefix"
                     ],
+                    "shell_env_params": [
+                        "experiment_id",
+                        "trial_id",
+                        "fio_prefix"
```

Comment:
As mentioned in #568, this gets a little cumbersome I think at times to have to declare a parameter as necessary or exposed throughout the full stack of the configs.
Also a conversation for a separate PR, but would be nice to have a way to say, all the children in this CompositeEnv (maybe just this one and not recursively, though I'm not sure about that), should have this arg required or this shell_env exposed.

Reply:
Then it could happen at the top of this stack and not have to be repeated after that.

Reply:
agree. I'll think of an elegant way to implement it